docs/page/css/bootstrap.min.css

```diff
@@ -0,0 +1,5 @@
+/*!
+ * Bootstrap v3.3.4 (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
```

Comment:
I believe we should mention this dependency in the `NOTICE` file and add a copy of the license to `licenses/LICENSE.bootstrap`.

Reply:
Sure.